consensus/reactor_test.go

```diff
@@ -131,6 +132,62 @@ func TestValidatorSetChanges(t *testing.T) {
 	// TODO: test more changes!
 }
 
+// Check we can make blocks with skip_timeout_commit=false
+func TestReactorWithTimeoutCommit(t *testing.T) {
```

Comment:
fixed in https://github.com/tendermint/tendermint/pull/351/commits/e46f14be896a739868028db2780b2ff0d07cff6d